dotcom-rendering/src/web/components/OnwardsData.tsx

```diff
@@ -18,6 +21,24 @@ type OnwardsResponse = {
 	description: string;
 };
 
+const fixedHeight = css`
+	height: 300px;
+`;
+
+const revealStyles = css`
+	/* We're using classnames here because we add and remove these classes
+	   using plain javascript */
+	.reveal {
+		animation: ${keyframes`
+			0% { opacity: 0; }
+			100% { opacity: 1; }
+		`} 4s ease-out;
+	}
+	.pending {
+		display: none;
+	}
+`;
```

Comment:
This is currently duplicated here but once this (and other PRs) are merged then we will have a use case for setting this css higher up, perhaps at the page level.